### ISSUE TITLE
QM COS01 epic

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/oslc-specs#432]
**Stage-specific**

- [ ] Reinstate QM shape checks https://github.com/oslc-op/oslc-specs/pull/431

**Checklist**

TBA

**Comment by berezovskyi** [comment]
Duplicate of #302

**Comment by berezovskyi** [comment]
QM COS01 on hold due to a missing SoU: #477

**Comment by berezovskyi** [comment]
https://docs.oasis-open-projects.org/oslc-op/am/v3.0/architecture-management-spec.html

**Comment by berezovskyi** [comment]
Tagged de73534c2cc2abd671ac7cb8d000a03b2eadfe5e as OS

**Comment by berezovskyi** [comment]
PR for the files is published under https://github.com/oslc-op/website/pull/308

**Comment by berezovskyi** [comment]
![image](https://user-images.githubusercontent.com/64734/162209338-2899cb85-ce99-4b51-a82f-162e0c3ae7ff.png)

All conneg tests pass, will re-run once the https://github.com/oslc-op/website/pull/308 is merged @jamsden
